### DISCUSSION TITLE
How to config Yourls from Http to https

### DISCUSSION THREAD

**Discussion by fritzImages** [YOURLS/YOURLS#2885] [Q&A]
i was able to enable the https certificate and updated yourls config.php to enable the https url.  Problem solved.  ( I forgot to save file config.php after my edits)...

I just purchased a rapidssl certificate for my Yourls domain http://fritzimag.es.  I have changed the Word Press Yourls plugin settings: YOURLS Custom URL domain from http://fritzimag.es to https://fritzimag.es (and ran api status=ok).  I also changed user/config.php to define( 'YOURLS_SITE', 'https://fritzimag.es' );. However when I create a new WP post, the short code for Yourls remains as http://fritzimag.es.   grrr.    I have validated that the SSL for new domain https://fritzimag.es and it works fine.

I am not sure what I need to do next,   any suggestions ?  Thanks a lot...

**Comment by ozh** [comment]
All YOURLS needs is that your config.php states that your YOURLS instance is on `https://sho.rt/` instead of `http://sho.rt/`

What you describe is certainly a WP plugin problem, to be seen with the plugin author I guess
